lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1227,6 +1228,13 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'If `true`, disables [Sidekiq instrumentation](/docs/agents/ruby-agent/background-jobs/sidekiq-instrumentation).'
         },
+        :disable_roda_auto_middleware => {
+          :default => false,
+          :public => true,
+          :type => Boolean,
+          :allowed_from_server => false,
+          :description => 'If `true`, disables agent middleware for Roda. This middleware is responsible for advanced feature support such as [page load timing](/docs/browser/new-relic-browser/getting-started/new-relic-browser) and [error collection](/docs/apm/applications-menu/events/view-apm-error-analytics).'
+        },
```

Comment:
After discussing `disable_middleware_tracing` this morning, I'm curious if Roda instrumentation will work if `disable_middleware_tracing` is set to `true`, but `disable_roda_auto_middleware` is set to `false`. Has this come up in your research so far?

Reply:
All seems to be working fine if `disable_middleware_tracing` is true, but `disable_roda_auto_middleware` is set to false. [This test](https://github.com/newrelic/newrelic-ruby-agent/pull/2144/files#diff-aeeae0fa20827c88507de124359d0418691e7f4bf94b2ef6ae335f3ff95d3889R114-R121) demonstrates. I also spun up a test app to be certain—happy to demo live ◡̈